domain-services/fiscal-event-service/src/main/java/org/egov/config/FiscalEventConfiguration.java

```diff
@@ -55,4 +55,7 @@ public void initialize() {
 
     @Value("${ifix.master.project.search.path}")
     private String ifixMasterProjectSearchPath;
+
+    @Value("${bulk.push.fiscal.event}")
+    private String bulkPushFiscalEventDataSize;
```

Comment:
Is this the max number of fiscal events supported per request? 
Please give a better env var name.

Reply:
is it fine `bulk.push.fiscal.events.size` ?

Reply:
Name it: `fiscal.event.request.max.size`
Make similar change to the java variable name as well.